input/fsh/SD_Genomics.fsh

```diff
@@ -71,37 +71,39 @@ Description: "The result of a tumor marker test. Tumor marker tests are generall
 * category 2..*
 * category contains mcode-category 1..1
 * category[mcode-category] = SCT#250724005 // Tumor marker measurement (procedure)
-// sushi error >>> * category[mcode-category].coding.version = ""  // make sure version is not specified
 * subject only Reference(CancerPatient)
-* subject ^definition = "Patient whose tumor marker test is recorded."
+* subject ^definition = "Patient whose test result is recorded."
 * effective[x] only dateTime or Period
 * value[x] only Quantity or Ratio or string or CodeableConcept
 // Already MS in US Core Obs Lab: status, category, code, subject, effective[x], value[x], dataAbsentReason
-* specimen MS
+* specimen only Reference(HumanSpecimen)
+* specimen MS  // is not MS in US Core 4.0.0 and 5.0.1 
+// RelatedCondition added 11/14/2022, see https://chat.fhir.org/#narrow/stream/229074-CodeX/topic/Reference.20between.20tumor.20characteristics.20and.20cancer.20diagnosis
+* extension contains RelatedCondition named relatedCondition 0..* MS 
+* extension[relatedCondition] ^short = "Condition associated with this test."
+* extension[relatedCondition] ^definition = "Associates the tumor marker test with a condition, if one exists. Condition can be given by a reference or a code. In the case of a screening test such as prostate-specific antigen (PSA), there may be no existing condition to reference."
 
-
-RuleSet: CancerRelatedSpecimenRules
+Profile: HumanSpecimen
+Parent: Specimen
+Id: mcode-human-specimen
+Title:  "Human Specimen Profile"
+Description:  "A specimen taken from a Patient. The profile includes extensions to capture a related condition, a more precise body site, and an identifier of source body structure at that site (for example, a tumor)."
```

Comment:
Just a minor comment to remove "related condition" in the description since the extension was removed from the profile.